sphinx_gallery/gen_gallery.py

```diff
@@ -284,6 +284,30 @@ def call_memory(func):
     except (ImportError, ValueError):
         pass
 
+    # Check for srcset hidpi images
+    srcset = gallery_conf["image_srcset"]
+    if not isinstance(srcset, (list, tuple)):
+        raise ConfigError(
+            "image_srcset must be a list of strings with the "
+            'multiplicative factor followed by an "x", '
+            'e.g. ["2.0x", "1.5x"]'
+        )
+    srcset_mult_facs = set()
+    for st in srcset:
+        if not isinstance(st, str) or st[-1:] not in ("", "x"):
+            raise ConfigError(
+                f"Invalid value for image_srcset parameter: {st!r}. "
+                "Must be a list of strings with the multiplicative "
+                'factor followed by an "x".  e.g. ["2.0x", "1.5x"]'
+            )
+        if st == "":
```

Comment:
@QuLogic was just looking at our check config func - why do we want to allow item in `gallery_conf["image_srcset"]` to be an empty string?

Reply:
That preserved the previous code; I think it's always been that way since its initial PR, but as of https://github.com/sphinx-gallery/sphinx-gallery/pull/808#issuecomment-823736082, I think that was dropped from the documentation or usage, but the config handling part never did.

Reply:
Ah thank you :pray: , I have a vague memory of empty string being a config option previously now.